backend/licenses-core/src/main/java/org/eclipse/sw360/licenses/db/LicenseDatabaseHandler.java

```diff
@@ -1284,4 +1284,12 @@ private String buildObligationText(JSONObject jsonObject, int level) {
         }
         return obligationText.replaceFirst("\n","");
     }
+
+    public List<License> quickSearchLicenseByShortName(String searchText) {
+        return licenseRepository.searchByShortName(searchText);
+    }
+
+    public List<License> quickSearchLicenseByName(String searchText) {
+        return licenseRepository.searchByName(searchText);
```

Comment:
Let's not cause confusion by calling these functions **quickSearch**. Please just call them `searchByShortName` and `searchByName` as they are not doing anything different.